notification/app/notification/models/android/Keys.scala

```diff
@@ -28,4 +29,6 @@ object Keys {
   final val AwayTeamScore = "AWAY_TEAM_SCORE"
   final val Topics = "topics"
   final val UniqueIdentifier = "uniqueIdentifier"
+  final val electoralVotesAvailable = "electoralVotesAvailable"
```

Comment:
@davidfurey you had a good suggestion to change this to `"electoralCollegeSize"` can we still do that? Also nitpick: should be `Electoral...` to match the names of the other keys.